Add line chart example that uses templates, legends, and annotations.

domoritz opened the pull request.

domoritz committed:
Add line chart example that uses templates, legends, and annotations.

domoritz committed:
Remove version, better colors

domoritz committed:
minor

arvind commented:
This looks great @domoritz. Is it ready to merge? One comment, I didn't know what "GDR" meant until I looked it up. "East Germany" may be more familiar to general users, or perhaps "German Democratic Republic (East Germany)" or some such variant?

domoritz committed:
Improve Falkensee example, use square symbol, use opacity for annotation, move y axis to back, describe GDR, better aspect ratio.

domoritz commented:
Thanks @arvind. I added the explanation and made some more improvements (use square symbols in legend, move y axis back, added opacity, improved aspect ratio) that I came up with over dinner. 

<img width="589" alt="screen shot 2016-01-20 at 20 38 33" src="https://cloud.githubusercontent.com/assets/589034/12471782/cdb1103c-bfb5-11e5-994a-6bf013ea40e3.png">
